modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -39,16 +44,17 @@ Note: When multiple partitions are built parallel, it needs more RAM and mandate
 
 === Failovers 
 
-During failover of Search service nodes, there is no partition movement, and hence the failover-rebalance is instantaneous. Search service promotes the replica index partitions to primary so that those serve the live cluster traffic instantly. Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. 
+During failover of FTS service nodes, there is no partition movement, and hence the failover-rebalance is instantaneous. Search service promotes the replica index partitions to primary so that those serve the live cluster traffic instantly. Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. 
  
```

Comment:
Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration.  -> We should rewrite this sentence. It's not clear if patch time is more straightforward or rebalance is shorter? This sentence needs a bit more simplification

Reply:
Please feel free to propose new contents./wordings..

Reply:
Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. -> Users can use failover and recovery rebalance while applying patches or upgrading software/hardware.